libpod/util_linux.go

```diff
@@ -91,3 +92,19 @@ func GetV1CGroups(excludes []string) cgroups.Hierarchy {
 		return filtered, nil
 	}
 }
+
+// LabelVolumePath takes a mount path for a volume and gives it an
+// selinux label of either shared or not
+func LabelVolumePath(path string, shared bool) error {
+	_, mountLabel, err := label.InitLabels([]string{})
+	if err != nil {
+		return errors.Wrapf(err, "error getting default mountlabels")
+	}
+	if err := label.ReleaseLabel(mountLabel); err != nil {
+		return errors.Wrapf(err, "error releasing label %q", mountLabel)
+	}
+	if err := label.Relabel(path, mountLabel, shared); err != nil {
+		return errors.Wrapf(err, "error setting selinux label for %s to %q", path, mountLabel)
```

Comment:
might be good to work the shared value into the error here, in case it make a difference.